app/helpers/form_helper.rb

```diff
@@ -442,6 +442,15 @@ def form_select_f(f, attr, array, select_options = {}, html_options = {})
     include_blank = select_options.delete(:include_blank)
     if include_blank
       addClass html_options, "include_blank"
+      blank_value = include_blank.is_a?(TrueClass) ? nil : include_blank
+      if array.kind_of?(Array) # incase array is options_for_select
+        array = array.to_a.dup
```

Comment:
We shouldn't need the to_a since array is already an Array, right?

Reply:
Yes, but now that I checked some things I think it will probably be best to check if array can do `.to_a` since I was trying to avoid errors with safebuffer, but Organization::ActiveRecord_Relation does have a `.to_a`  option